### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by wln19** [gpcv-liujin/Ada-MVS#3]
Thank you very much for the code you provided. I used your source code for training. At the 8th epoch, the training loss and test loss suddenly increased, causing the validation set prediction depth to be wrong. How did you solve this problem?

**Comment by gpcv-liujin** [comment]
> Thank you very much for the code you provided. I used your source code for training. At the 8th epoch, the training loss and test loss suddenly increased, causing the validation set prediction depth to be wrong. How did you solve this problem?

Sometimes an inappropriate learning rate may lead to instability during the training. Maybe you can stop the training process, adjusting the learning rate, and then resuming the training from the epoch just before the anomaly occurred.
